.eslintrc

```diff
@@ -11,5 +11,31 @@
   "rules": {
     "no-console": 1, // Means warning
     "prettier/prettier": 2 // Means error
-  }
+  },
+  "overrides": [
+    {
+      "files": ["**/*.ts"],
+      "env": { "browser": true, "es6": true, "node": true },
+      "extends": [
+        "eslint:recommended",
+        "plugin:@typescript-eslint/eslint-recommended",
+        "plugin:@typescript-eslint/recommended"
```

Comment:
we should should use the prettier here in `plugins` and `extends`, so we don't duplicate style rules.

Reply:
I have made it consistent with the javascript part using pettier